### PR TITLE
Julia command line version updated

### PR DESCRIPTION
**Pull request by cvazquezlos**

### CODE REVIEW

**Comment by tkelman** [comment]
the commit won't be right there.

**Comment by cvazquezlos** [comment]
Where then?

File: doc/manual/getting-started.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ command line::
       (_)     | (_) (_)    |  Documentation: http://docs.julialang.org
        _ _   _| |_  __ _   |  Type "?help" for help.
       | | | | | | |/ _` |  |
-      | | |_| | | | (_| |  |  Version 0.5.0-dev+2440 (2016-02-01 02:22 UTC)
+      | | |_| | | | (_| |  |  Version 0.5.0 (2016-09-19 18:14 UTC)
      _/ |\__'_|_|_|\__'_|  |  Commit 2bb94d6 (11 days old master)
```
Comment (tkelman) [inline doc/manual/getting-started.rst]: this commit, and branch - replace the entire output so it's consistent

**Comment by KristofferC** [comment]
Updated this to 0.6 release. Planning on merging tomorrow unless I hear something else.

**Comment by tkelman** [comment]
wrong branch.

**Comment by KristofferC** [comment]
Couldn't I just change the branch target?

**Comment by tkelman** [comment]
Would make a mess and need rebasing. Easier as a new pr.

**Pull request closed without merging**